### DISCUSSION TITLE
TypeError

### DISCUSSION THREAD

**Discussion by Mikikoko** [MuiseDestiny/zotero-reference#187] [Q&A]
![image](https://github.com/MuiseDestiny/zotero-reference/assets/106791667/b1b71b94-657f-44e9-a6a9-49dc623b2d1f)

更新zotero7之后也更新了reference 但是一直使用不了   看了后台报错情况如上图所示   请大佬们看看

**Comment by MuiseDestiny** [comment]
https://github.com/MuiseDestiny/zotero-reference/releases/tag/0.6.1

**Reply by Mikikoko** [reply]
感谢   已解决
